hooks/post_gen_project.py

```diff
@@ -1,15 +1,15 @@
-#!/bin/sh
+#!/bin/env python
```

Comment:
This doesn't run in an environment, so on some systems the default python might still point to python2

```suggestion
#!/bin/env python3
```